.github/workflows/build.yaml

```diff
@@ -375,3 +375,38 @@ jobs:
 
       - name: Run codespell
         run: codespell -q 7 -S ".git,build*,./src/third_party/*" -I misc/codespell-allowlist.txt
+
+  deploy:
+    name: Deploy
+    runs-on: windows-2019
+    strategy:
+        fail-fast: false
+    steps:
+      - name: Get source
+        uses: actions/checkout@v2
+
+      - name: Prepare Windows environment (Visual Studio)
+        if: runner.os == 'Windows'
+        uses: ilammy/msvc-dev-cmd@v1.5.0
+        with:
+          arch: x64
+
+      - name: Build and test
+        id: build-and-test
+        env:
+          CC: cl
+          CMAKE_GENERATOR: Ninja
+          CMAKE_PARAMS: -DCMAKE_BUILD_TYPE=Release -DZSTD_FROM_INTERNET=ON -DHIREDIS_FROM_INTERNET=ON -DENABLE_TESTING=OFF
+          CXX: cl
+        run: |
+          rc=0
+          ci/build || rc=$?
+          echo "::set-output name=exit_status::$rc"
+          exit $rc
+
+      - name: archive
+        uses: actions/upload-artifact@v2
+        if: success()
```

Comment:
The condition isn't needed as the build shouldn't reach this stage if there's a build error.